apps/common-lib/src/main/scala/com/gu/typerighter/controllers/PandaAuthController.scala

```diff
@@ -24,7 +24,6 @@ abstract class PandaAuthController(
     with HMACAuthActions
     with BaseController
     with Loggable {
-  override val apiGracePeriod: Long = 2 * 60 * 60 * 1000 // two hours worth of milliseconds
```

Comment:
This value is removed in PR https://github.com/guardian/pan-domain-authentication/pull/185 because
```
We want to move towards enforcing this pattern consistently across tools, so we are setting the default within the library to 24 hours.
```